server/routes/api.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express')
+const router = express.Router()
+const path = require('path')
+
+
+router.use(express.json())
+router.use(express.urlencoded({ extended: false }))
+
+
+const Transaction = require('../models/Transaction')
+
+router.post('/transaction',async function(req,res)
+{
+    let transaction = await new Transaction(req.body)
+    transaction.save()
+    res.send("saved successfully")
+})
+
+router.delete('/transaction',async (req,res) =>
+{
+    const transaction = req.body
```

Comment:
you could just send the `id` as param, instead of sending the whole transaction to the `req.body`